src/Identity/test/Identity.Test/CdnScriptTaghelperTests.cs

```diff
@@ -23,7 +24,9 @@ public CdnScriptTagTests(ITestOutputHelper output)
             _output = output;
         }
 
-        [Fact]
+        // Because this test runs on .NET Core and seems reliable there, likely not worth running on .NET Framework.
+        [ConditionalFact]
+        [FrameworkSkipCondition(RuntimeFrameworks.CLR, SkipReason = "At least flaky on .NET Framework.")]
```

Comment:
@Pilchie @dotnet/aspnet-build thoughts about filing a follow-up issue❔ I'm leaning toward no because the test is about external resources. Also realized the test runs fine on Linux and macOS.
![image](https://user-images.githubusercontent.com/6431421/114117089-cc01fd00-989a-11eb-8908-6fa359b21ab7.png)

Reply:
I think we should at least file an issue about figuring out _why_ the test failed

Reply:
If that's in dotnet/runtime, so be it

Reply:
@wtgodbe I agree and wasn't really questioning that. My question was more whether we needed an issue about removing the `[FrameworkSkipCondition(...)]`. Hearing none 😃 

@karelz @scalablecory where should I file an issue about the .NET Framework `HttpClient` class and the underlying SSL libraries❔

Reply:
@dougbu I believe we do not run dotnet/runtime tests on .NET Framework anymore. Before we stopped to run them, we saw quite some flakiness on .NET Framework in networking. I wonder if this is similar ...
If you're able to isolate it to HttpClient/SslStream repro, we can take a look -- I would start email thread with our team as a starting point.